components/Modal/Questionnaire/index.vue

```diff
@@ -0,0 +1,135 @@
+l<template>
+  <div id="questionnaire" class="questionnaire">
+    <div class="questionnaire__title">
+      <h1>Kuisioner Desa Digital</h1>
+    </div>
+    <div
+      v-if="showContent"
+      class="questionnaire__content"
+    >
+      <div class="questionnaire__content-text">
+        <p class="questionnaire__content-subtitle">
+          Desa Anda Masuk Kategori
+        </p>
+        <p class="questionnaire__content-village-level">
+          {{ village }}
+        </p>
+        <p class="questionnaire__content-info">
+          Agar desa Anda mendapatkan manfaat lebih besar. Yuk daftarkan desa Anda ke Desa Digital!
+        </p>
+      </div>
+      <div class="questionnaire__content-image">
+        <img
+          width="379px"
+          height="332px"
+          :src="villageImage"
+          class="questionnaire__content-image"
+          alt="Desa Digital"
+        >
+      </div>
+    </div>
+    <div class="questionnaire__action">
+      <BaseButton
+        class="questionnaire__action-signup"
+        label="Daftar Sekarang"
+      />
+      <nuxt-link
+        to="/"
+        class="questionnaire__action-cancel"
+      >
+        Nanti Saja
+      </nuxt-link>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    villageTypes: {
+      type: Array,
+      default: () => []
+    },
+    chosenLevel: {
+      type: Number,
+      default: null
+    }
+  },
+  data () {
+    return {
+      village: '',
+      villageImage: ''
+    }
+  },
+  computed: {
+    showContent () {
+      if (Array.isArray(this.villageTypes)) {
+        this.showVillage()
+        return true
+      } else {
+        return false
+      }
+    }
+  },
+  methods: {
+    showVillage () {
```

Comment:
`villageTypes` always `type array` on method `showVillage` because you have done check `type array` on method `showContent` i thing this is not good for me because it is redundant

Reply:
Okey then, do you have any suggestions about that kak @adzharamrullah ?

Reply:
remove one of it @doohanas

Reply:
thank you for your suggestion kak @adzharamrullah. You can see the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/80/commits/6363457b926ba87beb5762a20fbbbca2e37adc60)